{"name": "test_get_ner", "status": "broken", "statusDetails": {"message": "AttributeError: 'tuple' object has no attribute 'iterrows'", "trace": "self = <test_get_ner.TestNer object at 0x0000015FE2E20DD8>\n\n    @pytest.mark.nertest\n    @allure.feature(\"Ner实体识别\")\n    def test_get_ner(self):\n>       GetNer.get_ner(self, \"bio_char.txt\", \"bio_char_result.csv\", \"ner_test_result.csv\", \"tag.txt\")\n\napi\\test_get_ner.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\api\\get_ner.py:72: in get_ner\n    GetNer.get_ner_result(self, target_file, result_file)\n..\\api\\get_ner.py:42: in get_ner_result\n    lb_list1, lb_list2 = GetNer.get_data_list(self, data_file)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <test_get_ner.TestNer object at 0x0000015FE2E20DD8>\ntest_data_file = 'ner_test_result.csv'\n\n    def get_data_list(self, test_data_file):\n        test_data = ChangeDataType.ner_csv_to_dict(rootPath + \"\\\\testresults\\\\resultfile\\\\\" + test_data_file)\n        lb_list1 = []\n        lb_list2 = []\n>       for idx, temp in test_data.iterrows():\nE       AttributeError: 'tuple' object has no attribute 'iterrows'\n\n..\\api\\get_ner.py:26: AttributeError"}, "start": 1585728435649, "stop": 1585728625253, "uuid": "a28d6eb9-89a6-48a5-ae0d-f8117b9e1c8c", "historyId": "d8540c5ad7e84c36960538bd2f6c1a29", "testCaseId": "c9ca9e117f821440dd422ad8374b0348", "fullName": "pytest.api.test_get_ner.TestNer#test_get_ner", "labels": [{"name": "feature", "value": "Ner实体识别"}, {"name": "tag", "value": "nertest"}, {"name": "parentSuite", "value": "pytest.api"}, {"name": "suite", "value": "test_get_ner"}, {"name": "subSuite", "value": "TestNer"}, {"name": "host", "value": "WIN-OQ20V7UABJF"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest.api.test_get_ner"}]}